src/ImageProcessor/Imaging/Resizer.cs

```diff
@@ -234,44 +239,52 @@ protected virtual Bitmap ResizeLinear(Image source, int width, int height, Recta
 
                         upscale = true;
 
-                        switch (anchorPosition)
+                        if (anchorPoint.HasValue)
                         {
-                            case AnchorPosition.Left:
-                                destinationY = (height - sourceHeight) / 2;
-                                destinationX = 0;
-                                break;
-                            case AnchorPosition.Right:
-                                destinationY = (height - sourceHeight) / 2;
-                                destinationX = width - sourceWidth;
-                                break;
-                            case AnchorPosition.TopRight:
-                                destinationY = 0;
-                                destinationX = width - sourceWidth;
-                                break;
-                            case AnchorPosition.Top:
-                                destinationY = 0;
-                                destinationX = (width - sourceWidth) / 2;
-                                break;
-                            case AnchorPosition.TopLeft:
-                                destinationY = 0;
-                                destinationX = 0;
-                                break;
-                            case AnchorPosition.BottomRight:
-                                destinationY = height - sourceHeight;
-                                destinationX = width - sourceWidth;
-                                break;
-                            case AnchorPosition.Bottom:
-                                destinationY = height - sourceHeight;
-                                destinationX = (width - sourceWidth) / 2;
-                                break;
-                            case AnchorPosition.BottomLeft:
-                                destinationY = height - sourceHeight;
-                                destinationX = 0;
-                                break;
-                            default:
-                                destinationY = (height - sourceHeight) / 2;
-                                destinationX = (width - sourceWidth) / 2;
-                                break;
+                            destinationY = anchorPoint.Value.Y;
```

Comment:
There's no sanitation here for the X and Y values. We'll need that before we can continue.